pom.xml

```diff
@@ -253,27 +257,37 @@
             <goals>
               <goal>execute</goal>
             </goals>
-            <configuration>
-              <source>
-                import java.util.UUID
-                def uuidBackend = System.getProperty("uuid.hawkular.accounts.backend")
-                if (uuidBackend == null) {
-                    uuidBackend = UUID.randomUUID().toString()
-                    System.setProperty("uuid.hawkular.accounts.backend", uuidBackend)
-                }
-                def uuidUi = System.getProperty("uuid.hawkular.ui")
-                if (uuidUi == null) {
-                    uuidUi = UUID.randomUUID().toString()
-                    System.setProperty("uuid.hawkular.ui", uuidUi)
-                }
-                print "Set uuid.hawkular.accounts.backend = " + uuidBackend
-                project.properties.setProperty('uuid.hawkular.accounts.backend', uuidBackend)
-                project.properties.setProperty('uuid.hawkular.ui', uuidUi)
-                project.properties.setProperty('uuid.hawkular.secretstore', uuidUi)
-              </source>
-            </configuration>
           </execution>
         </executions>
+        <configuration>
+          <scripts>
+            <script><![CDATA[
+              import java.util.UUID
+              def uuidBackend = System.getProperty("uuid.hawkular.accounts.backend")
+              if (uuidBackend == null) {
+                  uuidBackend = UUID.randomUUID().toString()
+                  System.setProperty("uuid.hawkular.accounts.backend", uuidBackend)
+              }
+              def uuidUi = System.getProperty("uuid.hawkular.ui")
+              if (uuidUi == null) {
+                  uuidUi = UUID.randomUUID().toString()
+                  System.setProperty("uuid.hawkular.ui", uuidUi)
+              }
+              print "Set uuid.hawkular.accounts.backend = " + uuidBackend
+              project.properties.setProperty('uuid.hawkular.accounts.backend', uuidBackend)
+              project.properties.setProperty('uuid.hawkular.ui', uuidUi)
+              project.properties.setProperty('uuid.hawkular.secretstore', uuidUi)
+            ]]></script>
+          </scripts>
+        </configuration>
+        <dependencies>
+          <dependency>
+            <groupId>org.codehaus.groovy</groupId>
+            <artifactId>groovy-all</artifactId>
+            <version>${version.org.codehaus.groovy}</version>
+            <scope>runtime</scope>
```

Comment:
`runtime` dependencies are not included in the war, right? IIRC, it's `runtime` for the time that the build runs, not for the time the application runs, but better ask the maven expert :)